src/sxmlc.c

```diff
@@ -334,13 +334,21 @@ int XMLNode_copy(XMLNode* dst, const XMLNode* src, int copy_children)
 
 	/* Tag */
 	if (src->tag != NULL) {
+	#ifdef _MSC_VER
+		dst->tag = _strdup(src->tag);

```

Comment:
Is `_strdup()` only defined in MS Visual Studio? What is the problem with the standard `strdup()`?

Reply:
The issue here is that Visual Studio 2019 refuses to compile several old standard c functions, because they are subject to buffer overrun and some similar issues. Unless you activate an option to turn these errors into warnings in each projects, you cannot compile them because VS2019 generates an error on each of these functions. And I'm not very motivated to do this because it's strongly not recommended. But indeed you're right, modify the macro may be enough if the parameters number is the same between the 2 functions. Good remark.